README.md

```diff
@@ -42,6 +42,22 @@ This TodoList app uses Azure DocumentDB spring boot starter and AngularJS to int
         <version>${azure.spring.boot.starter.version}</version>^M
     </dependency>
     ```
+    
+* Customize Angular JS
+In `app.js`, make following changes. The client leverages Azure AD library for JS to handle AAD authentication in single page application. The following snippet of code configures adal provider for your registered app.
```

Comment:
pls sepcify full path of app.js. 
It's better to add detail steps on how to get tenant, what is client id, how to get it